api/builtins_qlik/HelmChart.go

```diff
@@ -58,7 +59,11 @@ func (p *HelmChartPlugin) Generate() (resmap.ResMap, error) {
 		p.HelmHome = directory
 	}
 
-	if len(p.ChartHome) == 0 {
+	if p.ChartHome == "" && p.TmpChartHome != "" {
```

Comment:
where will  set the `tmpChartHome`
in our config it is fixed like this https://github.com/qlik-oss/qliksense-k8s/blob/03d900865cead634d62083749b8314759285ab62/transformers/qseokversion.yaml#L16

Reply:
I was planning to change that line to this:
```
tmpChartHome: .chartcache/qliksense-init

```